src/backend/TrafficCourts/Citizen.Service/Services/Impl/FormRecognizerService.cs

```diff
@@ -68,9 +68,9 @@ public async Task<AnalyzeResult> AnalyzeImageAsync(MemoryStream stream, Cancella
         DocumentAnalysisClient documentAnalysisClient = new(_endpoint, credential);
 
         using Activity? activity = Diagnostics.Source.StartActivity("Analyze Document");
-        activity?.AddBaggage("ModelId", "ViolationTicket_v2");
+        activity?.AddBaggage("ModelId", "ViolationTicket_v3");
 
-        AnalyzeDocumentOperation analyseDocumentOperation = await documentAnalysisClient.StartAnalyzeDocumentAsync("ViolationTicket_v2", stream, null, cancellationToken);
+        AnalyzeDocumentOperation analyseDocumentOperation = await documentAnalysisClient.StartAnalyzeDocumentAsync("ViolationTicket_v3", stream, null, cancellationToken);

```

Comment:
Would it make sense to move these to configuration?

Reply:
Great suggestion.  Done.